src/builder.ts

```diff
@@ -29,35 +29,46 @@ import { Publisher } from './publisher'
 import * as Typegen from './typegen'
 import {
   assertPhotonInContext,
-  GlobalComputedInputs,
   Index,
-  isEmptyObject,
-  LocalComputedInputs,
+  InputsConfig,
   lowerFirst,
+  InputConfig,
+  PrismaInputFieldName,
+  CollapseToValue,
 } from './utils'
-
-interface FieldPublisherConfig {
+import { NexusArgDef, NexusInputObjectTypeDef } from 'nexus/dist/core'
+import {
+  WithRequiredKeys,
+  capitalize,
+  isEmpty,
+  merge,
+  split,
+  fromEntries,
+} from '@re-do/utils'
```

Comment:
As Jason said, it would be great not to add any new dependency, and inline these if you need them

Reply:
From my response to Jason:

> Given the amount of work that went into this PR and the span over which that work took place, maintaining them in multiple places was getting burdensome so I just added the dependency. It's a tiny package, but if you have problems with it you could copy everything over in a future change.